### PR TITLE
Add ReadHTML to support loading HTML tables

### LINKED ISSUE #105: Support for ReadHTML
It would be useful to add `ReadHTML` support.

I am not sure if we should make it into the `dataframe` package, which will introduce the `golang/x/net/html` dependency.
A possibility is to add an `importer` package for it alongside the existing `dataframe` and `series`.

### PR DESCRIPTION
**Pull request by typeless**
The algorithm is mostly translated from https://github.com/pandas-dev/pandas/blob/master/pandas/io/html.py#L419.

Fixes #105

### CODE REVIEW

File: dataframe/html.go [hunk 1]
```diff
@@ -0,0 +1,134 @@
+package dataframe
```
Comment (kniren) [inline dataframe/html.go]: I would rather have this on the dataframe.go file

**Comment by kniren** [comment]
Hi! Thank you very much for your work @typeless. I will merge this issue once the comments are addressed.

File: dataframe/html.go (hunk 1 above)
Comment (typeless) [reply dataframe/html.go]: Updated.

**Comment by typeless** [comment]
@kniren any comment on this PR?

**Comment by kniren** [comment]
Hi, sorry to keep you waiting, but I still didn't have time to review this. I'll do another round of PRs as soon as I can.

**Comment by typeless** [comment]
No problem.

**Comment by kniren** [comment]
Better late than ever?

**Pull request merged**